charts/reporting-operator/templates/custom-resources/report-queries/pod-volume-request.yaml

```diff
@@ -0,0 +1,115 @@
+apiVersion: metering.openshift.io/v1alpha1
+kind: ReportGenerationQuery
+metadata:
+  name: "persistentvolumeclaim-request-raw"
+  labels:
+    operator-metering: "true"
+{{- block "extraMetadata" . }}
+{{- end }}
+spec:
+  reportDataSources:
+  - "persistentvolumeclaim-request-bytes"
+  columns:
+  - name: labels
+    type: map<string, string>
+    tableHidden: true
+  - name: persistentvolumeclaim
+    type: string
+    unit: kubernetes_persistentvolumeclaim
+  - name: persistentvolume
+    type: string
+    unit: kubernetes_persistentvolume
+  - name: namespace
+    type: string
+    unit: kubernetes_namespace
+  - name: storageclass
+    type: string
+    unit: kubernetes_storageclass
+  - name: volume_request_memory_bytes
+    type: double
+    unit: bytes
+  - name: timeprecision
+    type: double
+    unit: seconds
+  - name: volume_request_memory_byte_seconds
+    type: double
+    unit: byte_seconds
+  - name: timestamp
+    type: timestamp
+    unit: date
+
+  query: |
+      SELECT labels,
+      labels['persistentvolumeclaim'] as persistentvolumeclaim,
+      labels['volumename'] as persistentvolume,
+          labels['namespace'] as namespace,
+          labels['storageclass'] as storageclass,
+          amount as volume_request_memory_bytes,
+          timeprecision,
+          amount * timeprecision as volume_memory_byte_seconds,
```

Comment:
Needs to be ‘as volume_request_memory_byte_seconds’. Should fix CI.

Reply:
addressed in commit ec9c079d9e6a7f6538ab4d477b1f831ed729e44c